### PR TITLE
added images for Remix

### PR DESCRIPTION
**Pull request by antonionardella**
added images to the remix configuration with more information

### CODE REVIEW

File: documentation/docs/guide/evm/tooling.md [hunk 1]
```diff
@@ -52,6 +52,13 @@ If you wish to use additional EVM chains with Metamask, you can simply add more
 
 If you also want to use the [Remix IDE](https://remix.ethereum.org/) to deploy any regular Solidity Smart Contract, you should set the environment as **Injected Web3**, which should then connect with your MetaMask wallet.
 
+Click on the _Deploy & Run transactions_ button in the menu on the left
+Select `Injected Web3` from the `Environment` dropdown.
+![image](https://user-images.githubusercontent.com/7383572/146169413-fd0992e3-7c2d-4c66-bf84-8dd4f2f492a7.png)
```
Comment (lucas-tortora) [inline documentation/docs/guide/evm/tooling.md]:
```suggestion

[![Select Injected Web3 from the Environment dropdown](https://user-images.githubusercontent.com/7383572/146169413-fd0992e3-7c2d-4c66-bf84-8dd4f2f492a7.png)](https://user-images.githubusercontent.com/7383572/146169413-fd0992e3-7c2d-4c66-bf84-8dd4f2f492a7.png)

```
This way the user can click on the image to expand it.  The extra new lines add some spacing.

File: documentation/docs/guide/evm/tooling.md [hunk 2]
```diff
@@ -52,6 +52,13 @@ If you wish to use additional EVM chains with Metamask, you can simply add more
 
 If you also want to use the [Remix IDE](https://remix.ethereum.org/) to deploy any regular Solidity Smart Contract, you should set the environment as **Injected Web3**, which should then connect with your MetaMask wallet.
 
+Click on the _Deploy & Run transactions_ button in the menu on the left
+Select `Injected Web3` from the `Environment` dropdown.
+![image](https://user-images.githubusercontent.com/7383572/146169413-fd0992e3-7c2d-4c66-bf84-8dd4f2f492a7.png)  
+Metamask will ask to connect to Remix and once connected the `Environment` will be set to `Injected Web3` with the `Custom (1074) network`  
+![image](https://user-images.githubusercontent.com/7383572/146169653-fd692eab-6e74-4b17-8833-bd87dafc0ce2.png)
```
Comment (lucas-tortora) [inline documentation/docs/guide/evm/tooling.md]:
```suggestion

[![Environment will be set to Injected Web3](https://user-images.githubusercontent.com/7383572/146169653-fd692eab-6e74-4b17-8833-bd87dafc0ce2.png)](https://user-images.githubusercontent.com/7383572/146169653-fd692eab-6e74-4b17-8833-bd87dafc0ce2.png)

```

File: documentation/docs/guide/evm/tooling.md [hunk 3]
```diff
@@ -52,6 +52,13 @@ If you wish to use additional EVM chains with Metamask, you can simply add more
 
 If you also want to use the [Remix IDE](https://remix.ethereum.org/) to deploy any regular Solidity Smart Contract, you should set the environment as **Injected Web3**, which should then connect with your MetaMask wallet.
 
+Click on the _Deploy & Run transactions_ button in the menu on the left
+Select `Injected Web3` from the `Environment` dropdown.
+![image](https://user-images.githubusercontent.com/7383572/146169413-fd0992e3-7c2d-4c66-bf84-8dd4f2f492a7.png)  
+Metamask will ask to connect to Remix and once connected the `Environment` will be set to `Injected Web3` with the `Custom (1074) network`
```
Comment (lucas-tortora) [inline documentation/docs/guide/evm/tooling.md]:
```suggestion
Metamask will ask to connect to Remix and once connected the `Environment` will be set to `Injected Web3` with the `Custom (1074) network`.
```

File: documentation/docs/guide/evm/tooling.md [hunk 4]
```diff
@@ -52,6 +52,13 @@ If you wish to use additional EVM chains with Metamask, you can simply add more
 
 If you also want to use the [Remix IDE](https://remix.ethereum.org/) to deploy any regular Solidity Smart Contract, you should set the environment as **Injected Web3**, which should then connect with your MetaMask wallet.
 
+Click on the _Deploy & Run transactions_ button in the menu on the left
```
Comment (lucas-tortora) [inline documentation/docs/guide/evm/tooling.md]:
```suggestion
Click on the _Deploy & Run transactions_ button in the menu on the left and select `Injected Web3` from the `Environment` dropdown.
```

File: documentation/docs/guide/evm/tooling.md [hunk 5]
```diff
@@ -52,6 +52,13 @@ If you wish to use additional EVM chains with Metamask, you can simply add more
...
+Select `Injected Web3` from the `Environment` dropdown.
```
Comment (lucas-tortora) [inline documentation/docs/guide/evm/tooling.md]:
```suggestion
```

**Review by lucas-tortora — APPROVED** [review]
Great addition! Thanks! Just some formatting comments on my end :)

**Review by jorgemmsilva — APPROVED** [review]

**Pull request merged**